lua/ibl/scope_languages.lua

```diff
@@ -298,6 +298,27 @@ local M = {
     mlir = {
         region = true,
     },
+    nim = {
+        ["if"] = true,
+        case = true,
+        try =  true,
```

Comment:
Please remove the extra space to make CI happy

```suggestion
        try = true,
```

Reply:
Done.